setup.py

```diff
@@ -105,6 +105,7 @@ def txt_read(*paths):
         "GeoNodes": ["cartopy[plotting]", ],  # required for visualization of GeoArray and GeoSurface nodes
         "BAG": ["hydroffice.bag>=0.2.10", ],  # required by BAG plugin
         "OpenDAP": ["pydap", ],  # required by OpenDAP plugin
+        "ADIOS": ["adios>=1.9.1b19", ],  # required by ADIOS plugin
```

Comment:
> In other words, I don't want that people try to build HDF Compass with all the plugins on Windows, getting an error, have to google around about solutions, and finally finding that GitHub ticket where we discussing about the issue.

I totally understand that. This is why the main file of this repository, [`README.md` reads](https://github.com/HDFGroup/hdf-compass#development-environment):
- ADIOS Plugin (Linux/OSX only)

> I understand that there is a way of building it on Windows, but it is not main stream. 

and as soon as one can build it, it will just work and this repo needs no update.

> If you can give me an example of frozen HDF Compass that I can run on my testing machines, I am fine with leaving as is. Otherwise, I would modify the setup.

Let us move the freezing questions to the [discussion here](https://github.com/HDFGroup/hdf-compass/issues/155#issuecomment-235557366). Currently, you can just *not install* the adios python wrapper and nothing will happen beside a "ADIOS Plugin not loaded" terminal message.